_posts/2017-12-11-away-week-prague/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "The importance of face-to-face in a remote world"
+title-short: Face-to-Face
```

Comment:
I think the title needs to match the URL from an SEO perspective? (Let's decide on one of the two and then update the other to match)